packages/venia-concept/src/components/Checkout/shippingForm.js

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import { Form } from 'informed';
+import { array, bool, func, shape, string } from 'prop-types';
+
+import Button from 'src/components/Button';
+import Label from './label';
+import Select from 'src/components/Select';
+
+import classify from 'src/classify';
+import defaultClasses from './shippingForm.css';
+
+class ShippingForm extends Component {
+    static propTypes = {
+        availableShippingMethods: array,
+        cancel: func,
+        classes: shape({
+            body: string,
+            footer: string,
+            heading: string,
+            shippingMethod: string
+        }),
+        shippingMethod: string,
+        submit: func,
+        submitting: bool
+    };
+
+    constructor(...args) {
+        super(...args);
+
+        this.state = {
+            shippingMethod: this.props.shippingMethod
+        };
+    }
+
+    componentDidMount() {
+        // If we don't have a shipping method, default to the first available one.
+        const { availableShippingMethods, shippingMethod } = this.props;
+        if (!shippingMethod) {
+            this.modifyShippingMethod(availableShippingMethods[0]);
+        }
+    }
+
+    render() {
+        const {
+            availableShippingMethods,
+            classes,
+            shippingMethod,
+            submitting
+        } = this.props;
+
+        // TODO: fix this hack that gets around React warnings about items having unique keys.
+        // We have to add a 'value' prop due to the Select component's getItemKey function.
+        const selectableShippingMethods = availableShippingMethods.map(
```

Comment:
This doesn't look like a hack to me. One way or another, you have to map your objects to `<option>` elements that have the correct `value` attribute, and they might as well key off of `value`, since it'll be unique for each option.

Realistically though, we should either

1. deprecate our `Select` component in favor of the one from `informed`, or
2. change our `Select` component to render the one from `informed`

Don't have to do that in this PR, though. And you'd still have to map data to `<option>` elements.

Reply:
I dig that, yeah.

Without having looked at its API yet, I vote for using the one from `informed` until we have a legitimate need for wrapping it.